src/commonMain/kotlin/fr/acinq/bitcoin/Script.kt

```
@@ -600,12 +648,56 @@ public object Script {
             }
         }
 
+        public fun checkSignatureSchnorr(pubKey: ByteArray, sigBytes: ByteArray, @Suppress("UNUSED_PARAMETER") scriptCode: ByteArray, signatureVersion: Int): Boolean {
```

Why does this function throw exceptions instead of just returning `false` on error cases? Is it really necessary?